src/hooks/screen/useCreateChat.ts

```diff
@@ -4,8 +4,8 @@ import SimpleToast from 'react-native-simple-toast';
 import ChannelList from '../../database/schema/ChannelListSchema';
 import SignedMessageRepo from '../../service/repo/signedMessageRepo';
 import UserSchema from '../../database/schema/UserSchema';
-import useChatUtilsHook from '../core/chat/useChatUtilsHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
+import useChatUtilsHook, {AllowedGoToChatScreen} from '../core/chat/useChatUtilsHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been reordered and a new type `AllowedGoToChatScreen` has been imported from `useChatUtilsHook`. This change is fine as it enhances the type safety of the code.
```diff
- import useChatUtilsHook from '../core/chat/useChatUtilsHook';
- import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
+ import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
+ import useChatUtilsHook, {AllowedGoToChatScreen} from '../core/chat/useChatUtilsHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->